code/modules/mob/living/carbon/human/species/species_attack.dm

```diff
@@ -76,6 +76,11 @@
 	damage = 10.5
 	shredding = 1
 
+/datum/unarmed_attack/claws/monkey
```

Comment:
this isnt the attack you gave it, look again

and disable shredding that is a big mob thing

Reply:
I'm not sure what you mean by. "This isn't the attack you gave it."

Reply:
/datum/unarmed_attack/claws/leaper/monkey

not the same typepath